src/main/java/com/esri/samples/localserver/local_server_dynamic_workspace_raster/LocalServerDynamicWorkspaceRaster.java

```diff
@@ -0,0 +1,204 @@
+
+
+import java.io.File;
+import java.util.Arrays;
+
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.ArcGISMapImageLayer;
+import com.esri.arcgisruntime.layers.ArcGISMapImageSublayer;
+import com.esri.arcgisruntime.layers.RasterSublayerSource;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.localserver.DynamicWorkspace;
+import com.esri.arcgisruntime.localserver.LocalMapService;
+import com.esri.arcgisruntime.localserver.LocalServer;
+import com.esri.arcgisruntime.localserver.LocalServerStatus;
+import com.esri.arcgisruntime.localserver.LocalService;
+import com.esri.arcgisruntime.localserver.RasterWorkspace;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.Viewpoint;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.sun.javafx.css.FontFace;
+
+import javafx.application.Application;
+import javafx.geometry.Insets;
+import javafx.geometry.Pos;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.StackPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Font;
+import javafx.scene.text.FontWeight;
+import javafx.scene.text.TextAlignment;
+import javafx.stage.FileChooser;
+import javafx.stage.FileChooser.ExtensionFilter;
+import javafx.stage.Stage;
+
+public class LocalServerDynamicWorkspaceRaster extends Application {
+
+	private MapView mapView;
+	private static final LocalServer server = LocalServer.INSTANCE;
+	private LocalMapService localMapService;
+	private ArcGISMapImageSublayer newSL;
+	
+	
+	@Override
+	public void start(Stage stage) throws Exception {
+		
+		
+	    try {
+	        // create stack pane and application scene
+	        StackPane stackPane = new StackPane();
+	        Scene scene = new Scene(stackPane);
+	        scene.getStylesheets().add(getClass().getResource("/css/style.css").toExternalForm());
+
+	        // set title, size, and add scene to stage
+	        stage.setTitle("Dynamic workspaces: raster");
+	        stage.setWidth(800);
+	        stage.setHeight(700);
+	        stage.setScene(scene);
+	        stage.show();
+
+	        // create a control panel
+	        VBox vBoxControl = new VBox(6);
+	        vBoxControl.setMaxSize(240, 150);
+	        vBoxControl.getStyleClass().add("panel-region");
+	        vBoxControl.setStyle("-fx-background-color: rgba(0, 0, 0, 0.7);");
+	       
+	        // create the descriptive label
+	        Label description = new Label("This application shows how to create a dynamic workspace connection " +
+	                "to a raster folder and display raster data in a map. Click the " +
+	                "button to select a local raster and add it to the map.");
+	        description.setFont(new Font("veranda",12));
```

Comment:
Ok. I'll need to stretch the VBox down a bit to make sure all of the description is legible. Is that ok?